templates/zerver/help/configure-default-new-user-settings.md

```diff
@@ -14,7 +14,7 @@ preference settings, including the following:
 
 * Display settings, including:
     * Default view ([Recent topics](/help/recent-topics) vs. [All messages](/help/reading-strategies#all-messages))
-    * [Light mode vs. dark mode](/help/dark-theme)
+    * [Light mode vs. dark theme](/help/dark-theme)
```

Comment:
Here we use `Light mode` and in some other places we use `Day mode` as pointed in one of the other comments.